### DISCUSSION TITLE
Realm migration issue (One or more errors occurred. (A Ruleset object already exists with primary key property ShortName == '"osu"'))

### DISCUSSION THREAD

**Discussion by weajus** [ppy/osu#16651] [Q&A] [answered]
I dont what did happen?
[attach_me.zip](https://github.com/ppy/osu/files/7951331/attach_me.zip)
!

**Comment by peppy** [comment, marked as answer]
Thanks for reporting the issue, will check on this for you.

**Reply by peppy** [reply]
This will be fixed via #16652.

To fix your client, please close osu! then overwrite your `client.realm` (in the main osu! folder) with this one (unzip it first):

[client.realm.zip](https://github.com/ppy/osu/files/7951506/client.realm.zip)

This is a fixed up version of your database, so all your beatmaps and scores should return. Do note that you had some beatmaps from the karaoke ruleset which were causing issues (because you removed that ruleset from the game some time ago). These will no longer show up.

**Reply by weajus** [reply]
Thanks!!
